### PR TITLE
Add `application/x-keepass2` for KeePass database files

### PR DESCRIPTION
**Pull request by FliegendeWurst**

### CODE REVIEW

File: src/custom-types.json [hunk 1]
```diff
@@ -328,6 +328,12 @@
   "application/x-javascript": {
     "compressible": true
   },
+  "application/x-keepass2": {
+    "extensions": ["kdbx"],
+    "sources": [
+      "https://github.com/keepassx/keepassx"
```
Comment (dougwilson) [inline src/custom-types.json]: I'm not seeing the mime type listed on this page, but perhaps I am missing it. Can you guide me to where it is specified?

Comment (FliegendeWurst) [reply src/custom-types.json]: In this file: https://github.com/keepassx/keepassx/blob/master/share/linux/keepassx.xml

Comment (dougwilson) [reply src/custom-types.json]: Please update the link to the specific page which lists the mime type and the file extension.

Comment (dougwilson) [reply src/custom-types.json]: Thank you! My plane is about to take off now, so will do a full review later today.

**Pull request merged**